apps/datapuller/src/lib/sections.ts

```diff
@@ -106,6 +106,18 @@ const formatSection = (input: ClassSection) => {
         };
       }),
     })),
+    sectionAttributes: input.sectionAttributes?.map((attr) => ({
+      attribute: {
+        code: attr.attribute?.code || "",
+        description: attr.attribute?.description || "",
+        formalDescription: attr.attribute?.formalDescription || "",
+      },
+      value: {
+        code: attr.value?.code || "",
+        description: attr.value?.description || "",
+        formalDescription: attr.value?.formalDescription || "",
+      },
+    })),
```

Comment:
requirementDesignation should also be changed to be like this. Right now it's just a code which isn't readable.